src/store/customer/actions.js

```diff
@@ -259,8 +259,13 @@ export const applyGiftCard = ({commit}, payload) => {
     }`
   )
     .then(({data}) => {
-      let amount = (data.applyGiftCard && data.applyGiftCard.amount) || 0
-      if (data.errors[0].message.indexOf('jwt malformed') > -1) data.errors[0].message = 'The Code is invalid! Please try again!'
+      let amount = (data.applyGiftCard && data.applyGiftCard.amount) || 0,
+        message = _.get(data, 'errors[0].message')
+      if (typeof message === 'string') {
```

Comment:
if( typeof message === 'string' &&
message.indexOf('jwt malformed') > -1)
data.errors[0].message = 'The Code is invalid! Please try again!'